examples/sharp-immersed-boundary/flow-around-particle-sharp/flow-around-particle-sharp.prm

```diff
@@ -0,0 +1,175 @@
+    set grid arguments     = 2,1,1: -6,-4,-4 : 10,4,4 : true
+    set initial refinement = 0
+  end
+end
+
+#---------------------------------------------------
+# Boundary Conditions
+#---------------------------------------------------
+
+subsection boundary conditions
+  set number = 5
+  subsection bc 0
+    set id   = 0
+    set type = function
+    subsection u
+      set Function expression = 1
+    end
+    subsection v
+      set Function expression = 0
+    end
+    subsection w
+      set Function expression = 0
+    end
+  end
+  subsection bc 1
+    set id   = 2
+    set type = slip
+  end
+  subsection bc 2
+    set id   = 3
+    set type = slip
+  end
+  subsection bc 3
+    set id   = 4
+    set type = slip
+  end
+  subsection bc 4
+    set id   = 5
+    set type = slip
+  end
+end
+
+#---------------------------------------------------
+# IB particles
+#---------------------------------------------------
+
+subsection particles
+  set number of particles                     = 1
+  set assemble Navier-Stokes inside particles = false
+  subsection extrapolation function
+    set stencil order = 2
+    set length ratio  = 1
+  end
+  subsection local mesh refinement
+    set initial refinement                = 3
+    set refine mesh inside radius factor  = 0.75
+    set refine mesh outside radius factor = 2
+  end
+  subsection particle info 0
+    set type            = cylinder
+    set shape arguments = 0.5;1.0
+    subsection position
+      set Function expression = 0;0;0
+    end
+    subsection orientation
+      set Function expression = 1.5707963267948966;0;0
+    end
+  end
+end
+
+#---------------------------------------------------
+# Mesh Adaptation Control
+#---------------------------------------------------
+
+subsection mesh adaptation
+  set type                 = kelly
+  set max refinement level = 10
+end
+
+#---------------------------------------------------
+# Non-Linear Solver Control
+#---------------------------------------------------
+
+subsection non-linear solver
+  subsection fluid dynamics
+    set verbosity             = verbose
+    set tolerance             = 1e-6
+    set max iterations        = 10
+    set force rhs calculation = true
+  end
+end
+
+#---------------------------------------------------
+# Linear Solver Control
+#---------------------------------------------------
+
+subsection linear solver
+  subsection fluid dynamics
+    set verbosity                             = verbose
+    set method                                = gmres
+    set max iters                             = 1000
+    set relative residual                     = 1e-3
+    set minimum residual                      = 1e-12
+    set max krylov vectors                    = 100
+    set preconditioner                        = ilu
+    set ilu preconditioner fill               = 0
+    set ilu preconditioner absolute tolerance = 1e-12
+    set ilu preconditioner relative tolerance = 1.00
```

Comment:
```suggestion
```

Reply:
Removed because they're default entries, the two remaining are default too but left to keep the linear solver subsection in the .prm file